pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumbgppeeringpolicies.yaml

```diff
@@ -132,6 +132,18 @@ spec:
                               whole second.
                             format: duration
                             type: string
+                          gracefulRestartEnable:
+                            description: GracefulRestartEnable flag, when set enables
+                              graceful restart capability.
+                            type: boolean
+                          gracefulRestartRestartTime:
+                            description: "RestartTime is the estimated time it will
+                              take for the BGP session to be re-established with peer
+                              after a restart. After this period, peer will remove
+                              stale routes. This is described RFC 4724 section 4.2.
+                              \n Default is 120s if empty or zero."
```

Comment:
I wonder why our default for `RestartTime` is `120s`:
 - RFC 4724, section 4 says "A suggested default for the Restart Time is a value less than or equal to the HOLDTIME carried in the OPEN."
- We default HoldTime to `90s`
- GoBGP defaults `RestartTime` to the [HoldTime value](https://github.com/osrg/gobgp/blob/f8c6d24e32d71a7ea0273a51b574fd63559e2943/internal/pkg/config/default.go#L224)

Reply:
There was some discussion around this here : https://isovalent.slack.com/archives/C036D1MNKNG/p1682352752459779 

Conclusion was 120s default is in FRR and BIRD, so keeping things consistent with them.

Reply:
Got it, no major objections about that then.